app/views/comments/index.html.erb

```diff
@@ -10,23 +10,23 @@
 
   <meta property="og:type" content="article" />
   <meta property="og:title" content="Discussion of <%= @commentable.title %>" />
-  <meta property="og:description" content="<%= @commentable.description || "A DEV Comment" %>" />
-  <meta property="og:site_name" content="The DEV Community" />
+  <meta property="og:description" content="<%= @commentable.description || "#{ApplicationConfig["COMMUNITY_NAME"]} Comment" %>" />
+  <meta property="og:site_name" content="<%= community_qualified_name %>" />
   <meta name="twitter:site" content="@<%= SiteConfig.social_networks_handle %>">
   <meta name="twitter:creator" content="@<%= @user.twitter_username %>">
   <meta name="twitter:title" content="<%= @commentable.title %>">
-  <meta name="twitter:description" content="<%= @commentable.description || "A DEV Comment" %>">
+  <meta name="twitter:description" content="<%= @commentable.description || "#{ApplicationConfig["COMMUNITY_NAME"]} Comment" %>">
 
   <% if @root_comment.present? %>
-    <link rel="canonical" href="https://dev.to<%= @root_comment.path %>" />
-    <meta property="og:url" content="https://dev.to<%= @root_comment.path %>" />
+    <link rel="canonical" href="<%= app_url %><%= @root_comment.path %>" />
```

Comment:
similar to the one above: `comment_url(@root_comment)` though I think `comment_url` is not yet in master but in #6916 - https://github.com/thepracticaldev/dev.to/pull/6916/files#diff-127a850b7d7b20173ad3fba88f8a25e3

You can definitely add it in this PR, I'll rebase mine when this gets merged. Or you can simply use `app_url(@root_comment.path)`